files/etc/uci-defaults/99-nextsec-netifyd

```diff
@@ -0,0 +1,7 @@
+if [ -f /usr/lib/libnetify-flow-actions.so.0.0.0 ] && ! grep -q np-nfa /etc/netifyd.conf; then
+    (
+        echo
+        echo "[plugin_detections]"
+        echo "np-nfa = /usr/lib/libnetify-flow-actions.so.0.0.0"
```

Comment:
```suggestion
        echo "np-nfa = /usr/lib/libnetify-flow-actions.so.0"
```

Reply:
See comment above

Reply:
@filippocarletti already tested it with the link.